src/worker.js

```diff
@@ -128,29 +159,23 @@ async function onCallbackQuery(callbackQuery) {
  * Handle incoming Message
  * https://core.telegram.org/bots/api#message
  */
-const onMessage = async (message) => {
+const onMessage = async (message) =>
+{
   console.log(`Received message: ${message.text}`);
 
   // check if cooldown
   const isReady = cooldownFunction();
 
-  if (!isReady) {
+  if (!isReady)
+  {
     console.log(`Skipping, bot on cooldown`);
     return;
   }
 
-  // check if its a greeting or blacklisted word to save chatgpt calls
```

Comment:
How can we monitor rate limits? After real world use I will have a better feeling for how aggressively we should conserve API calls. But I need to see the logs.

Reply:
> How can we monitor rate limits? After real world use I will have a better feeling for how aggressively we should conserve API calls. But I need to see the logs.

We cannot store logs currently but it can be viewed in realtime on the workers dashboard.. we can try log storage when we have a DB but we sure need to preserve API.

the cooldown paired with limited callers saves it well right now